misc/config_tools/data/nuc11tnbi5/shared.xml

```diff
@@ -52,6 +52,32 @@
     <MISC_CFG>
       <GPU_SBDF>0x00000010</GPU_SBDF>
     </MISC_CFG>
+    <vuart_connections>
+      <vuart_connection>
+        <name>Connection_1</name>
+        <type>legacy</type>
+        <endpoint>
+          <vm_name>ACRN_Service_VM</vm_name>
+          <io_port>0x3F</io_port>
+        </endpoint>
+        <endpoint>
+          <vm_name>POST_STD_VM1</vm_name>
+          <io_port>0x3F</io_port>
+        </endpoint>
+      </vuart_connection>
+      <vuart_connection>
+        <name>Connection_2</name>
+        <type>pci</type>
+        <endpoint>
+          <vm_name>ACRN_Service_VM</vm_name>
+          <vbdf>00:10.1</vbdf>
```

Comment:
Should this be `00:10.0`? Without function 0 Linux will not recognize other functions of the same device.